node/pkg/p2p/p2p.go

```diff
@@ -760,24 +760,30 @@ func Run(params *RunParams) func(ctx context.Context) error {
 						gs := params.gst.Get()
 						if gs == nil {
 							// No valid guardian set yet - dropping heartbeat
-							logger.Log(params.components.SignedHeartbeatLogLevel, "skipping heartbeat - no guardian set",
-								zap.Any("value", s),
-								zap.String("from", envelope.GetFrom().String()))
+							if logger.Core().Enabled(params.components.SignedHeartbeatLogLevel) {
```

Comment:
The other checks around here use `logger.Level().Enabled(...)` - should these be the same?